docs/user-guide/misc/visualization.md

```diff
@@ -0,0 +1,57 @@
+# Visualization
+
+Data in a Polars `DataFrame` can be visualized using common visualization libraries.
+
+<div style="display:none">
+```python exec="on" result="text" session="user-guide/misc/visualization"
+--8<-- "python/user-guide/misc/visualization.py:setup"
+```
+</div>
+
+We start with the simple Iris dataset where we scan a CSV and then do a group-by on the `species` column and get the mean of the `petal_length`.
+
+{{code_block('user-guide/misc/visualization','dataframe',[])}}
+
+```python exec="on" session="user-guide/misc/visualization"
+--8<-- "python/user-guide/misc/visualization.py:dataframe"
+```
+
+## Matplotlib
+
+To create a bar chart we can pass columns of a `DataFrame` directly to Matplotlib as a `Series` for each column. Matplotlib does not have explicit support for Polars objects but Matplotlib can accept a Polars `Series` because it can iterate through each `Series` in a similar way to a Python `list`.
+
+{{code_block('user-guide/misc/visualization','matplotlib_show_plot',[])}}
+
+```python exec="on" session="user-guide/misc/visualization"
+--8<-- "python/user-guide/misc/visualization.py:matplotlib_make_plot"
+```
+
+## Seaborn, Plotly & Altair
+
+Seaborn, Plotly & Altair can accept a Polars `DataFrame` so we can use the standard syntax. These libraries work with data in a Polars `DataFrame` via the [dataframe interchange protocol](https://data-apis.org/dataframe-protocol/). For some charts these libraries may do an internal copy of the data to a Pandas `DataFrame`.
```

Comment:
the interchange protocol is a standardised way of accessing underlying data, but libraries still need some dataframe library to process it

if you pass a dataframe to seaborn or plotly, then for any kind of chart, they are going to be working with pandas internally

Reply:
Cheers @MarcoGorelli 

I was trying to be minimal with the text here as this material is hard to keep up-to-date. I thought it best to just address the question of whether the data would be copied, do you think we need more?

Reply:
How about just

```
Seaborn, Plotly & Altair can accept a Polars `DataFrame` by leveraging the [dataframe interchange protocol](https://data-apis.org/dataframe-protocol/), which offers zero-copy-conversion where possible.
```
?